interfaces/PA-App/src/assets/i18n/ru.json

```diff
@@ -1 +1,180 @@
+      "supply-username": "Ваше полное имя:"
+    },
+    "preprinted-qrcode": {
+      "error": "Отсканированный QR-код нельзя использовать, попробуйте другой.",
+      "info-qr": "Будет больше текста",
+      "introduction": "Сотрудник по оказанию помощи/волонтер отсканирует QR-код, чтобы связать вашу информацию и передать ее вам. Пожалуйста, храните свой физический QR-код в безопасном месте.",
+      "more-info-qr": "О QR-кодах",
+      "no": "Нет",
+      "question": "Есть ли у гуманитарного работника/волонтера предварительно распечатанный QR-код для сканирования?",
+      "result-ok": "Отсканированный QR-код теперь связан с вашей учетной записью и личной информацией. Пожалуйста, храните свой физический QR-код в безопасном месте.",
+      "start-scanner": "Запустить сканер",
+      "try-again": "Попробуйте снова",
+      "yes": "Да"
+    },
+    "registration-summary": {
+      "documents": "Обязательно подготовьте следующие документы:",
+      "no-validation-conclusion": "Спасибо. Ваша регистрация завершена.   <br><br>   В ближайшее время вы должны получить SMS/WhatsApp на указанный номер телефона, подтверждающий вашу регистрацию.   <br><br>   Вы будете проинформированы, если ваша заявка будет успешной в течение 7 дней.   <br><br>   Это последнее сообщение на данный момент.",
+      "qr-code": "Покажите этот QR-код сотруднику службы помощи, когда его спросят.",
+      "validation-conclusion": "Спасибо. Ваша регистрация завершена. Вы должны получить SMS/WhatsApp на указанный номер телефона. Это SMS/WhatsApp подтверждает вашу регистрацию.<br><br>В последующие недели сотрудник организации может связаться с вами, чтобы организовать встречу для проверки предоставленной вами информации. Сотрудник по оказанию помощи придет к вам на эту встречу."
+    },
+    "select-fsp": {
+      "attributes": {
+        "error": "Что-то пошло не так. Пожалуйста, попытайтесь еще раз.",
+        "introduction": "Для этого варианта нам также понадобится следующее:",
+        "success": "Спасибо. Ваша информация сохранена."
+      },
+      "conclusion": "Спасибо. Вы выбрали '{{name}}'.",
+      "introduction": "Пожалуйста, выберите способ получения помощи, если вы участвуете в программе."
+    },
+    "select-language": {
+      "introduction": {
+        "en": "Please select your language.",
+        "es": "Por favor seleccione su idioma",
+        "am_ET": "እባኮትን ቋንቋ ይምረጡ።",
+        "ar": "الرجاء اختيار لغتك",
+        "fr": "Choisissez votre langue, s'il vous plaît.",
+        "nl": "Kies jouw taal.",
+        "ny_MW": "Chonde sankhani chiyankhulo chanu.",
+        "om_ET": "Maaloo qooqa keessan filadhaa.",
+        "pt_BR": "Por favor, selecione seu idioma.",
+        "ru": "Пожалуйста, выберите ваш язык.",
+        "saq_KE": "Enjere tegelu ng'ituk ino.",
+        "ti": "",
+        "tl": "Pumili ng iyong wika.",
+        "tuv_KE": "Aarimatoi, tetheu akituk kon.",
+        "uk": "Будь ласка, виберіть свою мову.",
+        "in": "Pilih bahasa anda."
+      },
```

Comment:
These sentences only need to be in the `en.json`-file, so they can't get "out of sync" between all the different language-files.